charts/studio/templates/configmap-studio.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: studio
 data:
   ALLOWED_HOSTS: "*"
-  API_URL:  "http{{ if $.Values.studioBackend.ingress.tlsEnabled }}s{{ end }}://{{ .Values.studioBackend.ingress.host }}/"
-  UI_URL: "http{{ if $.Values.studioUi.ingress.tlsEnabled }}s{{ end }}://{{ .Values.studioUi.ingress.host }}/"
+  API_URL: "{{.Values.global.ingress.host }}/api"
```

Comment:
I am not pretty sure if API_URL should contain `/api` path. In studio code, and in studio-selfhosted I see that it is set as default:
https://github.com/iterative/studio-selfhosted/blob/ca74133612604abb22e449810574690f4bb1bf55/docker-compose/base.yaml#L62

Reply:
It's correct :+1: